reactioncommerce/reaction
# To fix issue 4414

The Tags should be used  from `Products.hashtags`   or `Catalog.product.tagIds`

Resolves #4414  
Impact: **minor **
Type: **bugfix**

## Issue
It is happening because of Customer view from Catalog collection uses tagIds array and Products collection use `hashtags` array.   

## Solution
Use both fields to form the **tags** object in container before passing data to component. 

## Breaking changes
none




## Testing
1.  deploy the code from the PR 
2. Start the app and  visit localhost:3000 Click the  Basic Reaction Product
3. On details screen u should see the Tags listed.

More detail for what each of these sections should include are available in our [Contributing Docs](https://docs.reactioncommerce.com/reaction-docs/master/contributing-to-reaction)
